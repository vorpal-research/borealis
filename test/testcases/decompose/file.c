#include <stdio.h>
#include <stdlib.h>

int main() {

    FILE* in = fopen("file.txt", "r");
    if(in) {
        fclose(in);
        return fgetc(in);
    } else return 0;
}
