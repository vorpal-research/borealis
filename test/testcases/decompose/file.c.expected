
[
	
	{
		"location" : 
		{
			"filename" : "test/testcases/misc/file.c",
			"loc" : 
			{
				"col" : 16,
				"line" : 9
			}
		},
		"type" : "ASR-01"
	}
]
